dnsapi/dns_transip.sh

```diff
@@ -139,6 +141,14 @@ _transip_setup() {
   _saveaccountconf_mutable TRANSIP_Username "$TRANSIP_Username"
   _saveaccountconf_mutable TRANSIP_Key_File "$TRANSIP_Key_File"
 
+  # download keyfile if it's an URL
+  if [ "$(echo "$TRANSIP_Key_File" | head -c8)" = "https://" ]; then
```

Comment:
use the `_startswith()` function

Reply:
i was searching for beginswith and couldn't find it... thanks for the pointer